app/controllers/markets_controller.rb

```diff
@@ -0,0 +1,68 @@
+class MarketsController < ApplicationController
+  def index
+    @markets = Market.all
+  end
+
+  def login
+    id = params.permit(:login_id)[:login_id].to_i
+
+    if (id <= Market.last.id && id > 0)
```

Comment:
[`between?`](http://ruby-doc.org/core-2.0.0/Comparable.html#method-i-between-3F) is yet another way to do this that might be a little quicker to comprehend.